### PR TITLE
feat(csp): allow shared assets

### PR DESCRIPTION
**Pull request by fiji-flo**
## Summary
Allow assers from https://mdn.github.io/shared-assets/ managed via GitHub pages in https://github.com/mdn/shared-assets

### CODE REVIEW

File: libs/constants/index.js [hunk 1]
```diff
@@ -152,14 +152,24 @@ export const CSP_DIRECTIVES = {
     "wikipedia.org",
     "upload.wikimedia.org",
 
+    // Shared assets.
```
Comment (caugner) [inline libs/constants/index.js:155]:
```suggestion
```

File: libs/constants/index.js [hunk 2]
```diff
@@ -152,14 +152,24 @@ export const CSP_DIRECTIVES = {
...
     "www.gstatic.com",
   ],
   "manifest-src": ["'self'"],
-  "media-src": ["'self'", "archive.org", "videos.cdn.mozilla.net"],
+  "media-src": [
+    "'self'",
+    "archive.org",
+    "videos.cdn.mozilla.net",
+
+    // Shared assets.
```
Comment (caugner) [inline libs/constants/index.js:170]:
```suggestion
```

**Review by caugner — APPROVED** [review]
I'm not sure we need those comments, but otherwise LGTM. 👍

**Review by bsmth — APPROVED** [review]
Thank you!

**Comment by bsmth** [comment]
Merging now, ty!

**Pull request merged**